packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -160,19 +162,22 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
   const isInLobbyOrConnecting = currentPage === 'lobby';
   const hasJoinedCall = !!(currentPage && hasJoinedCallFn(currentPage, currentCallState ?? 'None'));
   const showControlBar = isInLobbyOrConnecting || hasJoinedCall;
+  const showMobilePane = isMobile && (showChat || showPeople);
```

Comment:
`showMobilePane` is always used negated in the conditionals. Should we rename this `showCallComposite` instead?

Reply:
And then add a comment here: 

    // When `mobileView` is set, the chat and people pane take up the entire viewport